### DISCUSSION TITLE
Google signin support

### DISCUSSION THREAD

**Discussion by siccous** [Infisical/infisical#768] [General]
According to this [PR#535](https://github.com/Infisical/infisical/pull/535) Google signin should be supported.

But it doesn't seems to work, I've set up both the needed environment variables CLIENT_ID_GOOGLE and CLIENT_SECRET_GOOGLE but it doesn't change anything?

Am I missing any additional steps to get it work?

**Comment by dangtony98** [comment]
Hi @siccous!

We've implemented it but are releasing it in the next few days along with another authentication method.

You should expect it by Monday next week 😄
